agent/main.go

```diff
@@ -63,13 +74,19 @@ func main() {
 		log.G(ctx).WithError(errors.New("invalid argument")).Fatal("id not set")
 	}
 
+	// Ensure this process is a subreaper or else containers created via runc will
+	// not be its children.
+	if err := sys.SetSubreaper(1); err != nil {
```

Comment:
Can you use constant for `1` to make it clearer what exactly is it.

Reply:
Yeah, will do